core/src/main/java/io/grpc/transport/Http2ClientStream.java

```diff
@@ -132,7 +143,9 @@ protected void transportDataReceived(Buffer frame, boolean endOfStream) {
   }
 
   /**
-   * Called by transports for the terminal headers block on a stream.
+   * Called by subclasses for the terminal headers block on a stream.
```

Comment:
terminal trailer metadata on a stream.

More generally we should use the terms 'header metadata' and 'trailer metadata'